Tools/TestWebKitAPI/Tests/ios/ActionSheetTests.mm

```diff
@@ -110,11 +111,11 @@ static void setOverrideViewControllerForFullscreenPresentation(UIViewController
     [webView setNavigationDelegate:navigationDelegate.get()];
     [rootViewController presentViewController:webViewController.get() animated:NO completion:nil];
 
-    // Since TestWebKitAPI is not a UI application, +[UIViewController _viewControllerForFullScreenPresentationFromView:]
+    // Since TestWebKitAPI is not a UI application, -[UIView _wk_viewControllerForFullScreenPresentation]
     // returns nil. To ensure that we actually present the action sheet from the web view controller, we mock this for the
     // time being until https://webkit.org/b/175204 is fixed.
     setOverrideViewControllerForFullscreenPresentation(webViewController.get());
-    ClassMethodSwizzler swizzler([UIViewController class], @selector(_viewControllerForFullScreenPresentationFromView:), reinterpret_cast<IMP>(overrideViewControllerForFullscreenPresentation));
+    ClassMethodSwizzler swizzler([UIView class], @selector(_wk_viewControllerForFullScreenPresentation), reinterpret_cast<IMP>(overrideViewControllerForFullscreenPresentation));
```

Comment:
(Whoops — these need to be `InstanceMethodSwizzler` instead, I think)

Thanks for the review!